### PR TITLE
Adds GH container registry for release and manual release

### PR DESCRIPTION
**Pull request by helto4real**
## Proposed change

Adds the ability to push to GH container registry for release and manual release. We are already doing this for devbuilds now.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (which adds functionality to an existing integration)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [ ] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] The code compiles without warnings (code quality chek)
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/net-daemon/docs

### CODE REVIEW

File: .github/workflows/push_docker_manual.yml [hunk 1]
```diff
@@ -4,36 +4,38 @@ on:
...
         uses: docker/setup-buildx-action@v1
-      -
-        name: 🧰 Available platforms
+      - name: 🧰 Available platforms
         run: echo ${{ steps.buildx.outputs.platforms }}
 
       - name: 🔓 Login to docker hub
         run: docker login -u ${{ secrets.DOCKER_USERNAME }} -p ${{ secrets.DOCKER_TOKEN }}
-
+      - name: 🔓 Login to GitHub Container Registry
```
Comment (ludeeus) [inline .github/workflows/push_docker_manual.yml:33]: I would use the same action to log in to the dockerhub registry for consistency

File: .github/workflows/push_docker_manual.yml [hunk 2]
```diff
@@ -43,3 +45,4 @@ jobs:
             --file ./Dockerfile . \
             --compress \
             --tag "netdaemon/netdaemon:feature-${{ github.event.inputs.tag }}"
```
Comment (ludeeus) [inline .github/workflows/push_docker_manual.yml:47]:
```suggestion
            --tag "netdaemon/netdaemon:feature-${{ github.event.inputs.tag }}" \
```

File: .github/workflows/tags_docker.yml [hunk 3]
```diff
@@ -36,18 +36,21 @@ jobs:
...
+      - name: 🔧 Set up Docker Buildx
         id: buildx
         uses: docker/setup-buildx-action@v1
-      -
-        name: 🧰 Available platforms
+      - name: 🧰 Available platforms
         run: echo ${{ steps.buildx.outputs.platforms }}
       - name: 🔓 Login to docker hub
         run: docker login -u ${{ secrets.DOCKER_USERNAME }} -p ${{ secrets.DOCKER_TOKEN }}
+      - name: 🔓 Login to GitHub Container Registry
```
Comment (ludeeus) [inline .github/workflows/tags_docker.yml:48]: I would use the same action to log in to the dockerhub registry for consistency

File: .github/workflows/tags_docker.yml [hunk 4]
```diff
@@ -58,6 +61,7 @@ jobs:
             --compress \
             --tag netdaemon/netdaemon:latest \
             --tag netdaemon/netdaemon:${{ steps.version.outputs.version }}
```
Comment (ludeeus) [inline .github/workflows/tags_docker.yml:63]:
```suggestion
            --tag netdaemon/netdaemon:${{ steps.version.outputs.version }} \
```

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: .github/workflows/tags_docker.yml [hunk 5]
```diff
@@ -58,6 +61,7 @@ jobs:
...
+            --tag ghcr.io/net-daemon/netdaemon:${{ steps.version.outputs.version }}
```
Comment (BeeHiveJava) [inline .github/workflows/tags_docker.yml:64]: Should we also push a 'latest' tag here?

Comment (helto4real) [reply .github/workflows/tags_docker.yml:64]: hmm yea that would be a good idea.

File: .github/workflows/push_docker_manual.yml (hunk 1 above)
Comment (helto4real) [reply .github/workflows/push_docker_manual.yml:33]: yepp, missed that one in the dev one

**Review by ludeeus — APPROVED** [review]

**Pull request merged**